### PR TITLE
[ADD] module 'currency_exchange_rate_inverted'

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
This module applies the inverse method in the conversion between two currencies.

The inverse method is a calculation method that uses the inverse (reciprocal)
exchange rate for the multiplier and divisor when converting amounts from one
currency to another.

The exchange rate of the multiplier and divisor are the reciprocal,
or opposite, of each other.

The inverse method multiplies the foreign amount by the
exchange rate to calculate the company currency (also called domestic) amount.
# Configuration

To convert amounts from one currency to another using the inverse method,
you have to go to 'Accounting / Configuration / Miscellaneous / Currencies'.
 Then select the foreign currency that you wish to maintain inversion for
 and set the flag 'Inverted exchange rate'.

### CODE REVIEW

**Comment by jjscarafia** [comment]
We have the same need in Argentina but we solve in another way https://github.com/ingadhoc/odoo-addons/blob/8.0/base_currency_inverse_rate/models/res_currency.py.

I like this aproach, I will test it! (why runbot no available?)

**Comment by JordiBForgeFlow** [comment]
@jjscarafia You can test on runbot now. I have rebuilt it. http://3139034-337-db740e.runbot1.odoo-community.org/web?db=3139034-337-db740e-all

**Comment by andhit-r** [comment]
@jbeficent  what do you think about my last comment on #335? If it's okay with You I will do the combining.

**Comment by levkar** [comment]
this addon requires currency_rate_update to be adjusted as well.

**Comment by levkar** [comment]
@jbeficent

I couldn't get it work. I tried making USD inversed. Currency rate update didn't work as I expected. Later when I create an invoice in EUR and change currency on the invoice it converted from the normal rate. it always converts with normal rate (instead of inverted rate) whether it is selected to work inverted or not.

**Comment by andhit-r** [comment]
@levkar would You mind reviewing my PR #355? It has similar purpose with this PR but using different approach. Thanks in advance.

**Comment by jcdrubay** [comment]
It would be really nice to add to the res.currency object/views a field `inverted_rate` as a computed fields `= 1 / current rate.`

In Vietnam:
 - 1 USD = 20,000 VND
 - currency for companies is VND.
 - no one knows the value of 1 VND, but people know the exchange rate of 1 USD.

**Comment by gustavovalverde** [comment]
What happened with this? Not getting pulled?

File: currency_exchange_rate_inverted/models/res_currency.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# © 2015 Techrifiv Solutions Pte Ltd
+# © 2015 Statecraft Systems Pte Ltd
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class ResCurrency(models.Model):
+    _inherit = 'res.currency'
+
+    rate_inverted = fields.Boolean('Inverted exchange rate')
```
Comment (jcdrubay) [inline currency_exchange_rate_inverted/models/res_currency.py:12]: Could this be a company dependent field?

- Company A has currency USD; exchange rates VND/USD doesn't need to be inverted
- Company B has currency VND (1USD=22000VND); exchange rate USD/VND needs to be inverted

**Comment by jcdrubay** [comment]
I started the migration of the module to version 10 in #508
I have also added the integration with currency_rate_update.

We might want to close this PR for version 8 and switch the effort on v10 (#508 ).

**Comment by andhit-r** [comment]
Hi @jbeficent , i know this PR for old 8.0. Can we reopened it since this module already merge on v10 and i think there are a bunch of v8 user out there still need this module (include me). Or i will open new PR if you prefer.

Many thanks.

**Pull request closed without merging**